src/server/middleware/transpileSource.ts

```diff
@@ -0,0 +1,68 @@
+import assets from "../../assets.ts";
+import transform from "../../transform.ts";
+import { LRU } from "../../deps.ts";
+import { Middleware } from "../../types.ts";
+import { createURL } from "../request.ts";
+import { isDev, sourceDirectory } from "../../env.ts";
+import {
+  replaceFileExt,
+  resolveFileUrl,
+  ValidExtensions,
+} from "../../resolver.ts";
+import { resolveConfig, resolveImportMap } from "../../config.ts";
+
+export default async function transpileSource(): Promise<Middleware> {
```

Comment:
Rename these functions that return middleware to: `createTranspileSourceMiddleware` and return named functions like `transpileSourceMiddleware`.